qgym/envs/initial_mapping/initial_mapping.py

```diff
@@ -158,11 +156,9 @@ class InitialMapping(Environment[Dict[str, NDArray[np.int_]], NDArray[np.int_]])
 
     def __init__(  # pylint: disable=too-many-arguments
         self,
+        connection_graph: nx.Graph | ArrayLike | Gridspecs,
```

Comment:
Nice change, this is also liked by the pylinting as it reduces the amount of initialization variables.